addons/l10n_ar/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/argentina.html',
     'countries': ['ar'],
     'icon': '/base/static/img/country_flags/ar.png',
-    'version': '3.6',
```

Comment:
please add demo data
we remove it from the PR 16.0 
but will add value for FUN test here
https://gist.github.com/zaoral/559b8794173112ab2c281556522e5d29

Reply:
@smetl I already fixed the demo data for 18.0, if you want to add it to this PR, only git apply the patch file
you can find it here https://gist.github.com/zaoral/b96454d14866c3980c5b502609aeb2a9

cc @vbe-odoo

Reply:
@smetl Sorry I did not see your previous message. thank you for your feedback I will add it in w new PR